types/random-number/index.d.ts

```diff
@@ -0,0 +1,29 @@
+// Type definitions for random-number 0.0
+// Project: https://github.com/ashnur/random-number
+// Definitions by: OpenByteDev <https://github.com/OpenByteDev>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+// TypeScript Version: 2.2
+
+type Generator = (min?: number|null, max?: number|null, integer?: boolean|null) => number;
+
+interface UnparsedOptions {
+    min?: number|null;
+    max?: number|null;
+    integer?: boolean|null;
```

Comment:
Is there a reason these are potentially `null`?

Reply:
If one of these properties is `null` or `undefined` the library sets the value to a default.
There is maybe some edge case where you pass user inputs as arguments, which are `null` when the default should be used:

```ts
import rn = require('random-number');

function getNumberInput(message: string): number|null { }
function getBooleanInput(message: string): boolean|null { }

const num = rn({
    min: getNumberInput('min'),
    max: getNumberInput('max'),
    integer: getBooleanInput('boolean')
});
```

My goal was to match the library, so I included it, If you think it shouldn't be this way I have no problem removing it.

Reply:
Yeah, it's definitely a sensitive balance. I think it probably catches more being more conservative and disallowing `null`. If users end up using it, they can always make it more permissive.